metashare/api/migrations/0059_auto_20200220_1939.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.3 on 2020-02-20 19:39
```

Comment:
@wlonk I was going to rename this, but I'm actually going to backport it to `0051` since that's equivalent in terms of causing trouble with your migrations. :)

Reply:
I think you care about the naming and number of migrations more than I do; to me, the cost of changing them once they've been made and pushed is high, and the cost of having many of them and having them have opaque names is low. Would you like me to change my view of them? I am happy to, to try to make things better for you.

Reply:
I don't feel strongly, but I do think sensible names (not "auto") are a higher priority for me. I don't mind as much having lots of them.

Reply:
Sounds good. I'll prioritize good names.